### PR TITLE
LATIR-2 Added Icon component

### LINKED ISSUE #2: LATIR-2: Icon Component
As a developer,
whenever I need to display an Icon with just the file and a size,
I want to use the Icon component

Also replace the 'LogoIcon' component for this Icon component in the navbar. The logo is available in the google drive files. 

Icon component:

props: 

- size: ("tiny"|"small"|"medium"|"big"|"huge")
- src: string
- alt: string

Size:
- tiny: 16x16
- small: 32x32
- normal: 64x64
- big: 128x128
- huge: 256x256

### PR DESCRIPTION
**Pull request by cavac96**
Closes #2

### CODE REVIEW

File: src/components/Navbar/NarrowView/index.js [hunk 1]
```diff
@@ -19,7 +20,7 @@ const NarrowLogoContainer = styled.header`
 const NarrowLogo = ({ onClick=identity }) => {
     return <NarrowLogoContainer onClick={onClick} >
         {/**This is where the icon will be*/}
```
Comment (Jkierem) [inline src/components/Navbar/NarrowView/index.js]: Please remove this comment

**Review by Jkierem — CHANGES_REQUESTED** [review]

**Review by Jkierem — CHANGES_REQUESTED** [review]
The branch seems to have conflicts with development. Do a rebase first

File: src/components/Navbar/NarrowView/index.js (hunk 1 above)
Comment (cavac96) [reply src/components/Navbar/NarrowView/index.js]: Done

**Comment by cavac96** [comment]
Rebase done. @Jkierem

**Review by Jkierem — APPROVED** [review]
Awesome!😁

**Pull request merged**